packages/lesswrong/components/common/SimpleDate.jsx

```diff
@@ -1,16 +1,11 @@
-import { registerComponent } from 'meteor/vulcan:core';
+import { Components, registerComponent } from 'meteor/vulcan:core';
 import React from 'react';
-import moment from 'moment-timezone';
-import Tooltip from '@material-ui/core/Tooltip';
-import withTimezone from '../common/withTimezone';
 
 /// A simple date, with no special cases, like "Jan 1, 2020". Hover over to
 /// also see the time.
-const SimpleDate = ({date, timezone}) => {
-  return <Tooltip title={moment(new Date(date)).tz(timezone).format('LLL z')}>
-      <span>{moment(new Date(date)).format("MMM DD, YYYY")}</span>
-  </Tooltip>
+const SimpleDate = ({date}) => {
```

Comment:
I think it might make sense to just drop SimpleDate and instead just use FormatDate here, since AFAICT it's only used once